### ISSUE TITLE
Make -vanilla a first-class citizen just like -respawn and -fast

### ISSUE THREAD

**Issue by ioan-chera** [team-eternity/eternity#323]
In this context, a first-class citizen means that -vanilla will be:

1. retained active even when player starts a new game or warps
2. included in the menu (unless I'm wrong, both -respawn and -fast are somewhere in the menu)
3. easily changed in the console.

It seems to be pretty much in demand, and I also use it often when playing wads for vanilla DOOM.

**Comment by Altazimuth** [comment]
Good idea. Long-time coming. We need to deal with the fact -vanilla turns off full freelook though (you have to manually toggle it back on as it stays off even after you're done running w/ -vanilla).
